ReactAndroid/src/main/java/com/facebook/react/views/scroll/ReactHorizontalScrollViewManager.java

```diff
@@ -120,6 +123,14 @@ public void setOverScrollMode(ReactHorizontalScrollView view, String value) {
     view.setOverScrollMode(ReactScrollViewHelper.parseOverScrollMode(value));
   }
 
+  /**
+   * Enables/Disables nested scrolling
```

Comment:
No need for that comment since it is pretty self explanatory.

Reply:
983e69f